### DISCUSSION TITLE
Encrypting Podman Secrets?

### DISCUSSION THREAD

**Discussion by dmocek** [containers/podman#15985] [General]
This Podman page: https://podman.io/releases/2021/04/02/podman-release-v3.1.0.html states "Please note that the initial implementation of secrets does not encrypt secrets at rest - look for this in an upcoming release.".  I cannot find any information on whether or not this has been implemented (so I assume not), or when it is planned to be implemented.  Is this planned, and if so, is there a target release?

**Comment by mheon** [comment]
@ashley-cui PTAL

**Comment by dmocek** [comment]
Any comment on this?

**Reply by ashley-cui** [reply]
No movement on this so far. Still on our radar as something that we're interested in implementing, but at the current moment nothing has been planned for it.
